Geometry/GEMGeometryBuilder/data/v11/GEMSpecs.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<DDDefinition xmlns="http://www.cern.ch/cms/DDL" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cern.ch/cms/DDL ../../../DetectorDescription/Schema/DDLSchema.xsd">
```

Comment:
```suggestion
<DDDefinition>
```

Reply:
@ianna Please confirm the long DDDefinition lines are not needed anymore.

Reply:
Shall I make this change?
Also, I copied and modified this from another version of the GEMSpecs, should we also update these to remove the schemas?

Reply:
@watson-ij I think these schemas are obsolete and no longer needed. Please change them in this PR or a later one.

Reply:
Thanks, I'll make the changes in a new PR